OpenRA.Mods.Cnc/UtilityCommands/ImportTSMapCommand.cs

```diff
@@ -565,5 +566,50 @@ static void ReadLighting(Map map, IniFile file)
 				})));
 			}
 		}
+
+		/// <summary>
+		/// Replaces destroyable cliff templates with ramp tiles
+		/// </summary>
+		static void SetupDestroyableCliffs(Map map)
+		{
+			var heightOffsets = new int[] {
+				0, -1, -1, -1, -1, 0,
+				0, -1, -2, -2, -1, 0,
+				0, 0, 1, 1, 0, 0,
+				0, 0, 0, 0, 0, 0
+			};
+
+			foreach (var cell in map.AllCells)
+			{
+				var tile = map.Tiles[cell];
+				if (tile.Type == 717)
```

Comment:
717 is a magic number

Reply:
717 and 718 refer to specific tileset tiles from the Westwood Tiberian Sun game which are required for the amp converter. "Magic numbers" will always be required to be used during map conversion, refer the TS temperat.ini for these particular ones.

Reply:
I think he means the 717 should get a descriptive name.

```csharp
const int LeftDestructibleCliffTileIndex = 717;
if (tile.Type == LeftDestructibleCliffTileIndex)
```

See https://en.wikipedia.org/wiki/Magic_number_(programming)#Unnamed_numerical_constants

Reply:
There are going to be other cases that need this, like waterfalls, so I will set up an enum instead of individual constants.